### PR TITLE
test /apply page

### LINKED ISSUE #34: test /apply page
test all the fields load correctly.

it would be sick if the dropdowns could be tested as well, making sure stuff actually loads when you click on the dropdown

### PR DESCRIPTION
**Pull request by NovaSagittarii**
chrome no likely :(
halp

### CODE REVIEW

**Comment by NovaSagittarii** [comment]
https://github.com/nextauthjs/next-auth/discussions/2053 THIS MAY BE THE WAY TO THE LIGHT

**Review by NovaSagittarii — COMMENTED** [review]
*something* here broke vercel deployments ...... but itd be great if i didnt have to test everything

File: cypress.config.ts [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
   projectId: 'krfkwu',
   e2e: {
     baseUrl: 'http://localhost:3000/',
     setupNodeEvents(on, config) {
       // implement node event listeners here
+      // https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/server-communication__env-variables/cypress.config.js
+      config.env = config.env || {};
+      config.env.SESSION_TOKEN = process.env.CYPRESS_SESSION_TOKEN || "12345678-abcd-efed-cba9-876543242069";
+      console.log('extended config.env with process.env.{SESSION_TOKEN}');
+      return config;
```
Comment (NovaSagittarii) [inline cypress.config.ts:14]: load session token for cypress

File: cypress/fixtures/session.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "name": {
+    	"first": "NAME",
+    	"last": "SURNAME"
+    },
+    "email": "test@example.com",
+    "image": "https://---.---"
+  },
+  "expires": "3000-01-01T00:00:00.000Z",
+  "accessToken": "abcdefghijklmnopqrst"
+}
```
Comment (NovaSagittarii) [inline cypress/fixtures/session.json:12]: fake user data the auth stub gives

**Review by minsooerickim — APPROVED** [review]
looks good mr test engineer 🙏🏻🙏🏻🙏🏻

**Pull request merged**